core/includes/database/select.inc

```diff
@@ -972,14 +972,14 @@ class SelectQuery extends Query implements SelectQueryInterface {
    *
    * @var array
    */
-  protected $alterTags = array();
+  public $alterTags = array();
```

Comment:
Switching visibility doesn't seem right to me. Why that?

Reply:
Can't change the visibility of properties